src/journal/journald-server.c

```diff
@@ -82,48 +82,27 @@
 
 #define NOTIFY_SNDBUF_SIZE (8*1024*1024)
 
-static int determine_space_for(
-                Server *s,
-                JournalMetrics *metrics,
-                const char *path,
-                const char *name,
-                bool verbose,
-                bool patch_min_use,
-                uint64_t *available,
-                uint64_t *limit) {
 
-        uint64_t sum = 0, ss_avail, avail;
+static int determine_path_usage(Server *s, const char *path, uint64_t *used, uint64_t *free) {
```

Comment:
Also, in quite a bit of code we name parameters that are used to return stuff "ret_xyz", to indicate that. I think this would be a good idea here, too. I.e. name this ret_used, and ret_free